### ISSUE TITLE
Move iCubGenova01 related files

### ISSUE THREAD

**Issue by xEnVrE** [robotology-playground/realsense-holder-calibration#1]
This repository, and the related documentation in the `README.md`, hosts file associated to the `iCubGenova01` robot (check [here](https://github.com/robotology/realsense-holder-calibration/tree/main/src/calibration/app/conf)).

Maybe, we could move them in `robots-configuration`.

**Comment by pattacini** [comment]
I would honestly keep the material related to this task hosted here.
The fact that those files have been used with iCubGenova01 is somewhat accidental.

The maintainer of the robot iCubGenova01 could also decide to make a copy into robots-configuration, but we need to maintain the necessary files here too.

**Comment by xEnVrE** [comment]
Actually I find very useful to have them here but I though they were in the wrong place as robot-specific so I opened the issue to discuss the fact. I didn't want to "pollute" the repository with such files.

However, if it is ok to have them here, then I think we can close the issue :)

**Comment by pattacini** [comment]
>However, if it is ok to have them here,

Yeah, it's perfectly ok as I think baseline conf files are needed anyhow.
Probably, the name of the files could be changed to be more robot-independent (and at this point a copy into robots-configuration would be recommended), but this is a nitpick.
